core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_treeselect.scss

```diff
@@ -0,0 +1,98 @@
+@use "variables" as *;
+@import "common";
+
+.p-treeselect {
+    @extend #form-field-base;
+    padding: 0;
+    transition: background-color 0.15s, border-color 0.15s, box-shadow 0.15s;
+    &.p-disabled {
+        @extend #form-field-disabled;
+
+        .p-treeselect-trigger {
+            color: $color-palette-gray-500;
+        }
+    }
+}
+
+.p-treeselect:not(.p-disabled):hover {
+    @extend #form-field-hover;
+}
+
+.p-treeselect:not(.p-disabled).p-focus {
+    @extend #form-field-focus;
+    outline-offset: 0;
+    box-shadow: $shadow-xs;
+}
+
+.p-treeselect .p-treeselect-label {
+    padding: $spacing-1 $spacing-2;
+    transition: background-color 0.15s, border-color 0.15s, box-shadow 0.15s;
+}
+
+.p-treeselect .p-treeselect-label.p-placeholder {
+    color: $color-palette-gray-500;
+}
+
+.p-treeselect.p-treeselect-chip .p-treeselect-token {
+    padding: 0.375rem $spacing-1;
+    margin-right: $spacing-1;
+    background: $color-palette-primary-200;
+    border-radius: $border-radius-sm;
+    color: $color-palette-primary;
+    font-size: $font-size-sm;
+}
+
+.p-treeselect .p-treeselect-trigger {
+    @extend #field-trigger;
+}
+
+p-treeselect.ng-invalid.ng-dirty > .p-treeselect {
+    border-color: $error;
+}
```

Comment:
do we need the `>`?

Reply:
Not really, I fix it with nested syntax.

```
p-treeselect {
    &.ng-invalid{
        &.ng-dirty {
            .p-treeselect {
                border-color: $error;
            }
        }
    }
...
```